Popup/Popup.js

```diff
@@ -313,7 +313,7 @@ const PopupBase = kind({
 						onTap={onClose}
 						size={closeButtonSize}
 					/> : null}
-					{title ? <Heading className={css.title} marqueeDisabled>{title}</Heading> : null}
+					{title ? <Heading className={css.title} alignment={centered ? 'center' : null}>{title}</Heading> : null}
```

Comment:
From the API doc http://nebula.lgsvl.com/enact/agate/docs/develop/docs/modules/agate/Popup/#PopupBase.centered `centered` should only apply for the contests NOT title.
I have one more question. marqee looks good to me but I wonder why `marqueeDisabled` applied the previous PR by @stanca-pop-lgp https://github.com/enactjs/agate/pull/503/files

Reply:
@seunghoh At that time we thought that Title would have to be centered as well, but when Marquee is applied text aligns left. It's fine with me to revert that change. Thanks!